sdk/react/bottomOverlay.js

```diff
@@ -251,14 +251,13 @@ const BottomOverlay = createReactClass({
       if ((previousAnnouncing === 0 || currentAnnouncing - previousAnnouncing > accessibilityProgressDelay)
               && currentPercent !== VALUES.MAX_PROGRESS_PERCENT) {
         previousAnnouncing = currentAnnouncing;
-        return this._renderDefaultProgressBar(playedPercent, scrubberBarAccessibilityLabel)
       } else {
         if (Platform.OS === 'android' && currentPercent === VALUES.MAX_PROGRESS_PERCENT && previousAnnouncing !== 0) {
           AndroidAccessibility.announce(scrubberBarAccessibilityLabel);
           previousAnnouncing = 0
         }
-        return this._renderDefaultProgressBar(playedPercent, "")
       }
+      return this._renderDefaultProgressBar(playedPercent, scrubberBarAccessibilityLabel)
```

Comment:
Looks good, but I have one question: I see that early in case when 
'currentPercent === VALUES.MAX_PROGRESS_PERCENT' is true, the method _renderDefaultProgressBar is called with empty string, that prevent playing Talkback in the end of video. 
Could you clarify, did you check this case? (end of video)